dev-aws/kafka-shared-msk/account-identity/data_infra.tf

```diff
@@ -1,14 +1,14 @@
 # module "account_identity_di_addr_lookup" {
 #   source           = "../../../modules/tls-app"
-#   consume_topics   = [kafka_topic.address_lookup_analytics]
+#   consume_topics   = [kafka_topic.]
```

Comment:
I guess this shouldn't change, right?

Reply:
I will enable that commented out thing in the consecutive PR
It will need an import though, didn't want to mix this change with topic definition etc.